src/main/java/com/znsio/teswiz/steps/Hooks.java

```diff
@@ -24,7 +24,8 @@ public void beforeScenario(Scenario scenario) {
                                   scenario.getName()));
         LOGGER.info(String.format("Hooks: Running test %s on %s", testExecutionContext.getTestName(),
                                   Runner.getPlatform().name()));
-        testExecutionContext.addTestState(TEST_CONTEXT.SCREENSHOT_MANAGER, new ScreenShotManager());
+        if(!Runner.getPlatform().name().equalsIgnoreCase("api"))
```

Comment:
Any loops/conditions should always have {}

Reply:
updated